statsmodels/regression/mixed_linear_model.py

```diff
@@ -2382,6 +2382,20 @@ def summary(self, yname=None, xname_fe=None, xname_re=None,
         info["Model:"] = "MixedLM"
         if yname is None:
             yname = self.model.endog_names
+        param_names = self.model.data.param_names[:]
+        n_fe_params = len(self.fe_params)
```

Comment:
minor naming convention change: change prefix "n" to "k"
n_fe_params -> k_fe_params
and similar for other cases

(the module use k_fe, but k_fe_params is more explicit here)

we try to keep "n" prefix for observation related counts, and "k" prefix for variable or